airbyte-integrations/connectors/source-twilio/source_twilio/schemas/conversation_participants.json

```diff
@@ -0,0 +1,79 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Generated schema for Root",
+  "type": "object",
+  "additionalProperties": true,
+  "properties": {
+    "participants": {
```

Comment:
Hello @btkcodedev you want t retrieve the individual records from participants AND not the array. The schema must be the data inside the `participants`object.

Reply:
Thanks for specifying the area
Resolved